pkg/acceptance/bettertestspoc/config/config.go

```diff
@@ -127,6 +127,31 @@ func ConfigVariablesFromModel(t *testing.T, model ResourceModel) tfconfig.Variab
 	return variables
 }
 
+// ConfigVariablesFromModels can be used to create a list of objects that are referring to the same resource model.
+// It's useful when there's a need to create associations between objects of the same type in Snowflake.
+func ConfigVariablesFromModels(t *testing.T, variableName string, models ...ResourceModel) tfconfig.Variables {
```

Comment:
let's reuse the method above inside this method (they share almost the same implementation)

Reply:
Done